src/duke/task/Task.java

```diff
@@ -0,0 +1,58 @@
+package duke.task;
+
+/**
+ * Represents a task with a description.
+ */
+public class Task {
+    // Indicators
+    public static final String STATUS_TICK = "[" + "\u2713" + "]";
+    public static final String STATUS_CROSS = "[" + "\u2718" + "]";
+
+    private String description;
+    private boolean isDone;
+
+    /**
+     * Creates a new task with the specified description.
```

Comment:
JDoc guidelines suggest we should phrase constructors is like so :
Constructs....